src/modules/launcher/PowerLauncher/PowerLauncher.csproj

```diff
@@ -86,6 +86,7 @@
     <PackageReference Include="System.Data.OleDb" />
     <PackageReference Include="System.ServiceProcess.ServiceController" />
     <PackageReference Include="UnitsNet" />
+	<PackageReference Include="WPF-UI" />

```

Comment:
Nit: This needs spaces instead of tab.

Reply:
I've pushed a fix for this: https://github.com/microsoft/PowerToys/pull/28538/commits/a09d3962b13536eae49057a4cd2ebe4f6187dcc0